# HG changeset patch
# User Rik <rik@octave.org>
# Date 1421215599 28800
#      Tue Jan 13 22:06:39 2015 -0800
# Node ID e97c63531e76f3db7987263e23f22a561a8ed59b
# Parent  7f5006dfa2ebd8d8ca5c570e113b1f9704c45a97
doc: Periodic grammarcheck of documentation.

* func.txi: Use @noindent for examples in the middle of sentences.

* warning_ids.m: Aline @item in table.

diff --git a/doc/interpreter/func.txi b/doc/interpreter/func.txi
--- a/doc/interpreter/func.txi
+++ b/doc/interpreter/func.txi
@@ -1393,24 +1393,26 @@ Note that for performance reasons it is 
 Octave functions, rather than to define anonymous functions which wrap an
 existing function.  The integration of @code{sin (x)} is 5X faster if the code
 is written as
 
 @example
 quad (@@sin, 0, pi)
 @end example
 
+@noindent
 rather than using the anonymous function @code{@@(x) sin (x)}.  There are many
 operators which have functional equivalents that may be better choices than an
 anonymous function.  Instead of writing
 
 @example
 f = @@(x, y) x + y 
 @end example
 
+@noindent
 this should be coded as
 
 @example
 f = @@plus
 @end example
 
 @xref{Operator Overloading}, for a list of operators which also have a
 functional form.
diff --git a/scripts/miscellaneous/warning_ids.m b/scripts/miscellaneous/warning_ids.m
--- a/scripts/miscellaneous/warning_ids.m
+++ b/scripts/miscellaneous/warning_ids.m
@@ -294,17 +294,17 @@
 ## @item Octave:shadowed-function
 ## By default, the @code{Octave:shadowed-function} warning is enabled.
 ##
 ## @item Octave:single-quote-string
 ## Print warning if a single quote character is used to introduce a
 ## string constant.
 ## By default, the @code{Octave:single-quote-string} warning is disabled.
 ##
-## @item Octave:nearly-singular-matrix
+## @item  Octave:nearly-singular-matrix
 ## @itemx Octave:singular-matrix
 ## By default, the @code{Octave:nearly-singular-matrix} and
 ## @code{Octave:singular-matrix} warnings are enabled.
 ##
 ## @item Octave:sqrtm:SingularMatrix
 ## By default, the @code{Octave:sqrtm:SingularMatrix} warning is enabled.
 ##
 ## @item Octave:str-to-num
